### ISSUE TITLE
DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working

### ISSUE THREAD

**Issue by florian42** [spulec/moto#3035]
When running our test cases pytest shows the following warning for moto:

```
/usr/local/lib/python3.7/site-packages/moto/cloudformation/parsing.py:407
  /usr/local/lib/python3.7/site-packages/moto/cloudformation/parsing.py:407: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3,and in 3.9 it will stop working
    class ResourceMap(collections.Mapping):

-- Docs: https://docs.pytest.org/en/latest/warnings.html
```

I would expect no warnings to be shown for moto when executing my test suite.

Im using moto version 1.3.14.

**Comment by bblommers** [comment]
Hi @florian42, this was fixed in https://github.com/spulec/moto/pull/2624 - the latest development release (https://pypi.org/project/moto/1.3.15.dev882/) should not have any warnings at all.

**Comment by jmehnle** [comment]
This was fixed in the 1.3.15 release. Time to close this issue?
